littlepay/api/groups.py

```diff
@@ -105,6 +105,12 @@ def remove_concession_group(self, group_id) -> bool:
         endpoint = self.concession_groups_endpoint(group_id)
         return self._delete(endpoint)
 
+    def get_concession_group_linked_funding_sources(self, group_id) -> Generator[dict, None, None]:
```

Comment:
Should the first type variable on the type hint be `GroupFundingSourceResponse` instead of `dict`?

Reply:
Good catch. I rebased on `main` and fixed this in the rebase.